dockerfile

```diff
@@ -18,18 +18,30 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
     wget \
     && apt-get clean
 
-# dbt
-RUN python3 -m venv /usr/local/venv
-RUN /usr/local/venv/bin/pip install \
+# Installing multiple versions of dbt
```

Comment:
Do we need to update dockerfile-prs ? at the moment render envs are only installing dbt 1.4